src/common/components/sortItemsComponent.spec.ts

```diff
@@ -23,7 +23,7 @@ describe('Sort items test', () =>  {
       };
       angular.mock.module('miqStaticAssets.common');
       angular.mock.inject(($componentController) => {
-        sortItemsCtrl = $componentController('miqSortItems', {$element: angular.element()}, bindings);
+        sortItemsCtrl = $componentController('miqSortItems', {$element: angular.element}, bindings);
```

Comment:
this looks wrong, `angular.element` is a function, not an `$element`... maybe something like `angular.element('<div></div>')` if you just want a fake empty div?